corehq/apps/sms/messages.py

```diff
@@ -73,6 +78,12 @@
         "[username]', entering your requested username in place of [username]"),
     MSG_REGISTRATION_WELCOME_CASE: ugettext_noop("Thank you for registering with CommCareHQ."),
     MSG_REGISTRATION_WELCOME_MOBILE_WORKER: ugettext_noop("Thank you for registering with CommCareHQ."),
+    MSG_UPDATE_UNRECOGNIZED_ACTION: ugettext_noop('Error: Unrecognized update action provided.'),
+    MSG_UPDATE: ugettext_noop("Usage: #update <keyword>."),
```

Comment:
we should remove the period at the end of the sentence to avoid any confusion in the syntax